specs/core/0_beacon-chain.md

```diff
@@ -2428,11 +2428,9 @@ def process_transfer(state: BeaconState, transfer: Transfer) -> None:
     )
     # A transfer is valid in only one slot
     assert state.slot == transfer.slot
-    # Only withdrawn or not-yet-deposited accounts can transfer
-    assert (
-        get_current_epoch(state) >= state.validator_registry[transfer.sender].withdrawable_epoch or
-        state.validator_registry[transfer.sender].activation_epoch == FAR_FUTURE_EPOCH
-    )
+    # Verify sender and receipient are not active
+    assert not is_active_validator(state.validator_registry[transfer.sender])
```

Comment:
That's not going to work because senders are liable to being slashed between exiting and withdrawing so we don't want them to transfer their balance away.